release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1114,7 +1114,7 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * The ability to delete operands from the web console using the `delete all operands` option is currently disabled. It will be re-enabled in a future version of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2012120[BZ#2012120] and link:https://bugzilla.redhat.com/show_bug.cgi?id=2012971[BZ#2012971].
 
-
+* Currently, Support level information about your cluster on the *Home* -> *Overview* -> *Details* card currently indicates the *Service Level Agreement (SLA)* is `unknown`. The incorrect status will be fixed in a future version of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2048059[*BZ#2048059].
```

Comment:
so we have only this line to be added for cluster support in RH, right?

Reply:
That is correct.